<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageFilterExampleWidget</class>
 <widget class="QWidget" name="ImageFilterExampleWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>196</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4">
     <item>
      <widget class="QLabel" name="sourceImageLabel">
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Source Image:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="sourceImageComboBox">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QLabel" name="targetImageLabel">
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Target Image:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="targetImageComboBox">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QLabel" name="transformObjectLabel">
       <property name="maximumSize">
        <size>
         <width>170</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="text">
        <string>Apply transform to:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="transformObjectComboBox">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="startButton">
       <property name="text">
        <string>Start</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="progressBar">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
